_data/accessibility-tests/memorable-date.yml

```diff
@@ -0,0 +1,79 @@
+title: Memorable date
+component_status: pass
+test_items:
+# General tests
+  - summary: Fields have auto-populate capability.
+    summary_additional: When you tab into memorable date fields using a keyboard, there will be an option in the browser to auto populate commonly used personal data.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.5
```

Comment:
Resolved in [bd2d373](https://github.com/uswds/uswds-site/pull/2919/commits/bd2d37342dc685affd1c78329a84c35f75e97cbb) unless @amycole501 and @amyleadem have additional input for this!

Reply:
Deferring to @amycole501 and @alex-hull here. 

Note: We do mention autocomplete for [1.3.5 on the footer accessibility page](https://designsystem.digital.gov/components/footer/accessibility-tests/), but I am not sure I understand why we would consistently expect users to have autocomplete set up on memorable date. Maybe it is expected to use it for birth date (`autocomplete="bday"`)? 

Unless we have a clear expectation for what the autopopulate should be, I lean towards making this criterion more generic like @mahoneycm suggested.

Reply:
Memorable date is personal data (their birthday) wherein any data fields such as birth date and year can be auto-populated. It can be generic though since it's meant for anyone to benefit from not just screen reader users.

Reply:
This largely would come down to implementation. We could maybe add a note saying that _if_ it's use case is to collect personally data from a user, check for and use relevant [autocomplete values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values).

It appears that birthday related values are the only option for auto complete with dates but I would argue memorable date could be used for an array of other use cases.

I'm curious if we should update our example since it's using Date of Birth as the example pattern.

Reply:
@amyleadem @amycole501 @mahoneycm Should I stick with the recommended update that Charlie gave?

Reply:
@amycole501 and/or @alex-hull, 

Our [memorable date guidance](https://designsystem.digital.gov/components/memorable-date/) does not explicitly state that memorable date is intended for a specific use case like birth dates. Additionally, the only [native autocomplete values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) related to date are "birth date" and "credit card expiration date". 

Because of these two things, it feels like we can't require users to have autocomplete turned on in all instances of the component. And because of that, it feels like we need to re-write this test in order to make a reasonable checklist item. 

Possible solutions:
1. Make this a conditional test and re-write the summary to something like  "Memorable date components for birth dates should have autocomplete capability".
2. Make a more generic summary like @mahoneycm suggested and like we have on other components for 1.3.5.

Let us know what you prefer!

Reply:
I agree with generalizing it, so people don't feel stuck in it. I think like said above, our memorable date is not specific to one item or another. We could again, mark this as conditional, as it will depend on the implementation. I feel that if we leave this in there as a mandatory criteria, people will begin to feel restricted to this form of implementation.

Reply:
Added a question about the test status related to this generalized summary in [this comment](https://github.com/uswds/uswds-site/pull/2919#discussion_r1835106998).

Reply:
Looks like we've determined updating the summary to be more generalized like @mahoneycm suggested and keeping the test status as conditional. Summary has been updated in [bd2d373](https://github.com/uswds/uswds-site/pull/2919/commits/bd2d37342dc685affd1c78329a84c35f75e97cbb) and test status is already set as conditional so resolving this convo!